### PR TITLE
GitHub: Added Crowdin link to issue config

### LINKED ISSUE #12364: Translation: wording/translation of "window" and "pane" to Dutch is...difficult
### Description

In **Settings** > **General** > **Context menu**, there are four options. Two of them are in English and two are in Dutch. Number 2 = "Show option to open folders in a new window" and number 3 = "Toon optie om mappen in nieuw venster te openen". Considering the translation, the words _window_ and _venster_ are basically **identical**. The significant difference is between "window" and "pane", but in Dutch, the second word/meaning is close to the first.

image

Comparing to Windows Explorer: the words "detailvenster" and "voorbeeldvenster" are used (for details en preview, respectively)
image

### Steps To Reproduce

Not a real bug

### Requirements

Translation that reflects the different meanings, to be unambiguous. The actual menu entry uses "**deelvenster**", which is correct. For example: https://www.interglot.com/dictionary/en/nl/search?q=pane

### Files Version

2.4.71

### Windows Version

Windows 11

### Log file

[debug.log](https://github.com/files-community/Files/files/11486840/debug.log)

### PR DESCRIPTION
**Pull request by Jay-o-Way**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #12364

**Validation**
Approved via [comment](https://github.com/files-community/Files/issues/12364#issuecomment-1556062821)

### CODE REVIEW

**Comment by Jay-o-Way** [comment]
Anything else we could add?

**Review by 0x5bfa — APPROVED** [review]
LGTM. I didn't know that you can do that.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1 +1,5 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Translation issues
```
Comment (yaira2) [inline .github/ISSUE_TEMPLATE/config.yml:3]:
```suggestion
  - name: Translation issue
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1 +1,5 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Translation issue
+    url: https://crowdin.com/project/files-app
+    about: Please help translation on Crowdin.
```
Comment (yaira2) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: Help improve translations on Crowdin.
```

**Review by yaira2 — APPROVED** [review]
LGTM

**Review by 0x5bfa — APPROVED** [review]
LGTM

**Comment by hishitetsu** [comment]
Should we add a link to Crowdin on the About page of the settings dialog as well?

**Comment by yaira2** [comment]
Good idea, can you open an issue to track this?

**Pull request merged**